packages/hawtio/src/plugins/shared/jolokia-service.ts

```diff
@@ -215,17 +215,19 @@ class JolokiaService implements IJolokiaService {
     // Initialising Jolokia instance
     this.jolokia = this.createJolokia(jolokia => {
       // Checking versions
-      jolokia.version(
-        onVersionSuccessAndError(
-          (version: JolokiaResponseValue) => {
-            log.info('Jolokia version:', {
-              client: jolokia.CLIENT_VERSION,
-              agent: (version as VersionResponseValue).agent,
-            })
-          },
-          error => log.error('Failed to fetch Jolokia version:', error),
-        ),
-      )
+      jolokia
+        .version(
+          onVersionSuccessAndError(
+            (version: JolokiaResponseValue) => {
+              log.info('Jolokia version:', {
+                client: jolokia.CLIENT_VERSION,
+                agent: (version as VersionResponseValue).agent,
+              })
+            },
+            error => log.error('Failed to fetch Jolokia version:', error),
+          ),
+        )
+        .then(() => true)
```

Comment:
why do we need this?

Reply:
It was my silly way to get rid `Promise returned from request is ignored` warnings from IDEA.
But right - I think warning is less confusing than this `.then()`.